utils/process.py

```diff
@@ -0,0 +1,52 @@
+# stdlib
+import errno
+import os
+
+# 3p
+try:
+    import psutil
+except ImportError:
+    psutil = None
+
+# project
+from util import Platform
+
+
+def pid_exists(pid):
+    """
+    Check if a pid exists.
+    Lighter than psutil.pid_exists
+    """
+    if psutil:
+        return psutil.pid_exists(pid)
+
+    # We don't support windows for now
+    if Platform.is_windows():
+        raise NotImplementedError("pid_exists is not implemented for Windows")
```

Comment:
Nice! Thanks, I'll add that to the new function, and we can reuse it in `jmxfetch.py` cleanly.